app/src/main/res/layout-land/view_onboarding_page.xml

```diff
@@ -28,94 +28,102 @@
                 tools:src="@drawable/illustration_onboarding_explore" />
         </FrameLayout>
 
-        <LinearLayout
+        <androidx.core.widget.NestedScrollView
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_gravity="center"
             android:layout_weight="1"
-            android:orientation="vertical">
+            android:scrollbars="vertical">
```

Comment:
[image](https://user-images.githubusercontent.com/9540770/127254006-2c84f956-b286-4d42-bc5d-ca85e653de7e.png) Lgtm...! but only one minor thing is that the scrollbars are available even when the content is fully visible and there is no need for scrolling..

Reply:
Hmm, let's just remove the scroll bar, then. Done!